test/markdownlint-test.js

```diff
@@ -1032,6 +1032,44 @@ tape("configMultiple", (test) => {
     });
 });
 
+tape("configMultipleWithRequireResolveBefore", (test) => {
```

Comment:
I could not find alternatives to Jest’s `before()` / `after()` hooks, so followed the idea in https://github.com/substack/tape/issues/59#issuecomment-32809481

Reply:
I'm not sure the symlink will work on Windows. And introducing an ordering dependency here will cause problems later when switching to an async runner. Can we move this to a copy+delete in the single test function? I know if it throws an exception it won't clean up all the way, but that's fine with me.

Reply:
Symlink creation should work on Windows too. CI does not seem to complain.